cmd/test-client/main.go

```diff
@@ -167,13 +167,13 @@ func (o *GrpcProxyClientOptions) Validate() error {
 	}
 	if o.proxyUdsName != "" {
 		if o.proxyHost != "" {
-			return fmt.Errorf("please do set proxy host when using UDS")
+			return fmt.Errorf("please set proxy host to \"\" when using UDS")
```

Comment:
I see these zero values must be set due to defaulting, and it matches the [README instructions](https://github.com/kubernetes-sigs/apiserver-network-proxy/blob/master/README.md?plain=1#L157)

Thanks for improving!